_posts/2021-02-23-OpenSSL.markdown

```diff
@@ -0,0 +1,58 @@
+---
+title: Switching back to OpenSSL
+layout: post
+---
+
+The Void Linux team is planning on [performing the switch to
+OpenSSL](https://github.com/void-linux/void-packages/pull/21056) before March
+is over.
+
+For most users, this shouldn't be noticeable, since we will be pushing the same
+packages that already had to be rebuilt twice a year due to LibreSSL ABI
+breaks, but if anything in your setup depends explicitly on LibreSSL (you know
+who you are!), be sure to migrate away from that.
+
+The important parts of the post are done, but you can read on to learn about
+the decision process and some things we had to do.
+
+The discussion started in an
+[RFC](https://github.com/void-linux/void-packages/issues/20935) from April
+12th, 2020, and has been ongoing since then, with most maintainers expressing
+support for the idea. The main reasons for the switch are as listed in the RFC:
+
+- Less need for patching packaged software, which is made especially difficult
+  and potentially risky by the complexity of the OpenSSL API.
+- Better platform support for accelerated implementation on non `x86_64`
+  devices.
+- Access to new standards and algorithms earlier, such as full TLS 1.3 support.
+  Users of packages such as OpenVPN will benefit greatly from this.
```

Comment:
This sounds like an aside, and should probably be formatted as such.

Reply:
Yeah, it doesn't really say anything -- why does OpenVPN get a callout that nothing else does?

Reply:
OpenVPN had breakage in the past due to LibreSSL, which was noticed by end users. So using it as example isn't bad imo. Furthermore it is used by a lot of people.

Reply:
I think more detail about that would be a better example than just saying "it'll benefit" -- not everyone might remember the openvpn breakage.

Reply:
That's fair, I will add a link to it. I think it's important to mention, since it's a simple real world example, but it could be better explained.